tests/unit/helpers/data_set_helper.py

```diff
@@ -418,6 +419,103 @@ def CSDUP_add_group_stdout(data_set_name):
         DFH5109 I END OF DFHCSDUP UTILITY JOB. HIGHEST RETURN CODE WAS: 0
     """.format(data_set_name)
 
+def get_job_output_execution(rc=0):
+    job = get_sample_job_output()
+    return _execution(
+        name="Get job output for {0}".format(job["job_id"]),
+        rc=rc,
+        stdout=job.get("ret_code").get("msg"),
+        stderr=job.get("ret_code").get("msg_txt")
+    )
+
+def _get_job_dd_output(ddname, job_id):
```

Comment:
Why have we got some private methods and non private methods here? The underscores seem inconsistent